### PR TITLE
fix: allocated track not persisted (CMC-837 )

### PR DESCRIPTION
**Pull request by sabahirfan**
### JIRA link (if applicable) ###
[CMC-837](https://tools.hmcts.net/jira/browse/CMC-837)

### Change description ###
This PR fixes the bug of not persisting the allocatedTrack.

**Does this PR introduce a breaking change?** (check one with "x")

- [ ] Yes
- [x] No

### CODE REVIEW

**Review by swalker125 — DISMISSED** [review]
LGTM

File: e2e/fixtures/createClaim.js [hunk 1]
```diff
@@ -15,6 +15,7 @@ module.exports = {
       }
     },
     claimant: {
+      allocatedTrack: 'SMALL_CLAIM',
```
Comment (pd-dudkiewicz) [inline e2e/fixtures/createClaim.js]: We expect allocatedTrack to be set by service callback, so we shouldn't set it via data store request

**Review by pd-dudkiewicz — CHANGES_REQUESTED** [review]

File: e2e/fixtures/createClaim.js (hunk 1 above)
Comment (sabahirfan) [reply e2e/fixtures/createClaim.js:18]: As the same is being used by api tests, and we need this for apit test as well.

Comment (sabahirfan) [reply e2e/fixtures/createClaim.js]: done

**Review by pd-dudkiewicz — APPROVED** [review]

**Review by dharmendrak — APPROVED** [review]

**Pull request merged**